docs/skopeo-list-tags.1.md

```diff
@@ -72,6 +72,9 @@ This commands refers to repositories using a _transport_`:`_details_ format. The
         "docker.io/myuser/myimage:v1.0"
         "docker.io/myuser/myimage@sha256:f48c4cc192f4c3c6a069cb5cca6d0a9e34d6076ba7c214fd0cc3ca60e0af76bb"
 
+## Docker archive
```

Comment:
This should follow format of the `docker://` paragraph above, an in particular precisely document the accepted syntax.

Reply:
this had done